src/actions/index.js

```diff
@@ -1,83 +1,76 @@
+import axios from 'axios';
 import * as types from './types';
 import {
   firebase,
   applicationsRef,
-  firestore,
   rsvpRef,
 } from '../config/firebase';
 
+const API_PATH = 'https://slohacks-backend-api.herokuapp.com';
+
 export const signUp = (values, callback) => (dispatch) => {
   dispatch({ type: types.SIGN_UP_ATTEMPT });
-  firebase.auth().createUserWithEmailAndPassword(values.email, values.password)
-    .then((userCredential) => {
-      firebase.auth().currentUser.sendEmailVerification()
-        .then(() => {
-          dispatch({ type: types.SIGN_UP_GUCCI, userCredential });
-          callback();
-        }).catch(error => dispatch({ type: types.SIGN_UP_FAIL, error }));
-    }).catch((error) => {
-      dispatch({ type: types.SIGN_UP_FAIL, error });
+  axios
+    .post(`${API_PATH}/users/signup`, {
+      email: values.email,
+      password: values.password,
+    })
+    .then((response) => {
```

Comment:
We should refactor all of the promise chaining into async/await
https://gyazo.com/0cb798a96ce4b4b15b50a09eb3913c99